src/PrestaShopBundle/Twig/Extension/GridExtension.php

```diff
@@ -158,6 +161,32 @@ public function renderColumnHeader(array $column, array $grid)
         ]);
     }
 
+    /**
+     * @param array $grid
+     *
+     * @return bool
+     */
+    public function isOrderingColumn(array $grid)
+    {
+        if (empty($grid['columns'])
+            || empty($grid['sorting']['order_by'])
+            || empty($grid['sorting']['order_way'])
+            || 'asc' != strtolower($grid['sorting']['order_way'])) {
+            return false;
+        }
+
+        foreach ($grid['columns'] as $column) {
+            if ('position' == $column['type']) {dump($column);
```

Comment:
dont forget to remove `dump($column);` :)

Reply:
holly cow 😱

Reply:
done!